samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -31,8 +33,15 @@ public void initializeSdk(SampleApplication application) {
         // TODO: Remove old builder and use new builder only to initialize the SDK
         // The new method should be called after the old method till the old method is removed
         // This is because the push and in-app modules are still using properties only initialized in the old method
-        CustomerIO.Builder oldBuilder = new CustomerIO.Builder(sdkConfig.getSiteId(), BuildConfig.API_KEY, application);
+        io.customer.sdk.android.CustomerIO.Builder oldBuilder = new io.customer.sdk.android.CustomerIO.Builder(sdkConfig.getSiteId(), BuildConfig.API_KEY, application);
         oldBuilder.build();
+        // TODO: Remove this once push module is decoupled and started using EventBus
+        new Handler(Looper.getMainLooper()).postDelayed(() -> {
+            String token = io.customer.sdk.android.CustomerIO.instance().getRegisteredDeviceToken();
+            if (token != null) {
+                CustomerIO.instance().registerDeviceToken(token);
+            }
+        }, 3000);
```

Comment:
This is temporary workaround to register token using new implementation, once EventBus is introduced, we should no longer need this.

Reply:
LOL 😂